runtime/vm/jvminit.c

```diff
@@ -7896,3 +7910,119 @@ parseGlrOption(J9JavaVM* jvm, char* option)
 
 	return JNI_ERR;
 }
+
+static UDATA
+ensureHashedHashFn(void *entry, void *userData)
+{
+	J9UTF8 *hashtableEntry = *(J9UTF8 **)entry;
```

Comment:
The indentation in the new code does not look correct, I guess you are using spaces instead of tabs ?

Reply:
I converted spaces to tabs in the file but didn't see anything wrong with my additions, however I did find old code that used spaces so I changed those.